### PR TITLE
changed in layout.scss and programs.scss

### LINKED ISSUE #1229: Extra apace between Navbar and starting text of the page
### Description
There is Extra apace between Navbar and starting text of the page  . here are some locations 
http://127.0.0.1:4000/catalog/
http://127.0.0.1:4000/talks-and-presentations
http://127.0.0.1:4000/programs/
http://127.0.0.1:4000/subscribe
127.0.0.1:4000 avoid it 

### Expected Behavior
Should not be extra space there

### Screenshots

image
image
image
image

### Enviroment:

 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Rajivkumawat3**
**Description**

This PR fixes #1229

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
Hey @Rajivkumawat3 , please dont make another pull request instead commit on the existing ones.There are currently 2 another PR linked with the same issue

**Comment by Rajivkumawat3** [comment]
ok

**Comment by vishalvivekm** [comment]
@Rajivkumawat3 Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

File: _sass/layout.scss [hunk 1]
```diff
@@ -213,6 +213,8 @@ blockquote:before {
 
 .page {
     margin-bottom: 0;
+    padding-top: 0;
+    padding-bottom: 0;
```
[the remark this answers is not in the document]
Comment (Rajivkumawat3) [reply _sass/layout.scss:217]: Done sir

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Comment by thisiskaransgit** [comment]
@HarshDobariya79, the change you were referring to for the calendar section was incorporated here

**Comment by HarshDobariya79** [comment]
> @HarshDobariya79, the change you were referring to for the calendar section was incorporated here

Oh, right. Thank you @thisiskaransgit for pointing it out.

**Review by Chadha93 — APPROVED** [review]
Thank you gents

**Pull request merged**